sdk/python/feast/data_source.py

```diff
@@ -46,13 +46,22 @@ class SourceType(enum.Enum):
 class FileOptions:
     """
     DataSource File options used to source features from a file
+
+    args:
```

Comment:
Args should be in the `__init__`. You can have a look here https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
fixed